### PR TITLE
Shinigami.id

### LINKED ISSUE #15408: Reaper Scans (ID): changed their domain and name to shinigami.id
### Source information and language

Reaper Scans 1.4.39 (Multi) 

### Source new URL

https://shinigami.id/

### Other details

Reaper Scans change url and name

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated all installed extensions.
- [X] I have checked if the source URL is not already updated by opening WebView.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by mobi2002**
closes #15408

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/overrides/madara/reaperscans/src/ReaperScansFactory.kt [hunk 1]
```diff
@@ -49,7 +49,11 @@ class ReaperScansTr : ReaperScans(
     override val mangaDetailsSelectorTag = ""
 }
 
-class ReaperScansId : ReaperScans("https://reaperscans.id", "id") {
+class ReaperScansId : ReaperScans("https://shinigami.id", "id") {
```
Comment (alessandrojean) [inline multisrc/overrides/madara/reaperscans/src/ReaperScansFactory.kt:52]: As they're not part of Reaper Scans anymore as it seems, its actually better to remove it from the multi lang extension and make it a separate single lang extension.

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Review by alessandrojean — APPROVED** [review]

**Comment by Riztard** [comment]
so people need to install the new ext right? 
but no migrating, since it keeps the id

**Comment by alessandrojean** [comment]
Correct

**Comment by kataceking** [comment]
> so people need to install the new ext right? but no migrating, since it keeps the id

Maksudnya gimana bang

**Comment by JoyBoy666** [comment]
> so people need to install the new ext right? but no migrating, since it keeps the id

New ext nya blom ada ya kan bg ?

**Comment by Riztard** [comment]
> New ext nya blom ada ya kan bg ?

blom

> Maksudnya gimana bang

kalo udah di merge, nanti harus install ext yang baru biar bisa lagi

**Pull request merged**